content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -339,15 +340,23 @@ const result = prisma.$executeRawUnsafe(
 $executeRawUnsafe<T = unknown>(query: string, ...values: any[]): PrismaPromise<number>;
 ```
 
-#### Raw query type mapping
+### Raw query type mapping
 
 <Admonition type="warning">
 
 This feature is currently not available for SQLite.
 
 </Admonition>
 
-In Prisma 4.0.0 and later, database values returned from raw queries are mapped ((_!by whom?!_) their corresponding [JavaScript types](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures). This matches the behavior of regular Prisma query methods such as `findMany`.
+Prisma 4.0.0 and later maps database values returned from raw queries to their corresponding [JavaScript types](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures). This matches the behavior of regular Prisma query methods such as `findMany`.
+
+<Admonition type="info">
+
+Raw query type mapping was available from Prisma 3.14.0 with the preview feature `improvedQueryRaw`. It is now [Generally Available](/about/prisma/releases#generally-available-ga) from Prisma 4.0.0. You do not need to use `improvedQueryRaw` in Prisma 4.0.0 or later.
+
+</Admonition>
```

Comment:
This is nice, text wise.

Design wise I would personally prefer to avoid admonitions for this as they take a lot of space and ask for attention, although we generally try to avoid putting attention on this but just have it present for the people that in this weird spot of potentially requiring the preview feature flag present because of their version.

Reply:
I want to make it clear visually that this text is an aside, and that the main flow continues after it. This paragraph only relevant to readers interested in earlier versions. As a standard paragraph it breaks the flow too much. I've amended the text to signal more clearly that this is an aside.